action.yml

```diff
@@ -48,7 +54,8 @@ runs:
       changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"perf","section":"Performance Improvements","hidden":false}]'
       bump-minor-pre-major: true
       default-branch: ${{ inputs.default-branch }}
-  - run: npm publish --access public
+      prerelease: ${{ inputs.prerelease }}
+  - run: npm publish ${{ inputs.npm-dist-tag }} --access public
```

Comment:
Suggestion: `npm publish --tag ${{ inputs.npm-dist-tag }}` and set default to `latest`